docs/farming.md

```diff
@@ -43,36 +73,62 @@ You can find these executables in the [Releases](https://github.com/subspace/sub
 ```
 ./subspace-farmer-x86_64-*-snapshot farm
 ```
-7. It may prompt again in here if you are using macOS. Refer to the note on step 4.
+7. It may prompt again in here. Refer to the note on step 4.
+</p>
+</details>
 
+<details><summary> 🍎 macOS Instructions (Click to Expand)</summary>
+<p>
 
+### 📝 macOS Installation
 
-**Windows**
+1. Download the executables for your operating system from the [Releases](https://github.com/subspace/subspace/releases) tab.
+2. Open your favourite terminal, and change to the Downloads directory using `cd Downloads`
+3. Make the farmer & node executable  `chmod +x $(farmer-name)` & `chmod +X $(node-name)`
+4. We will then start the node using the following command
+>(replace `INSERT_YOUR_ID` with a nickname you choose)
+
+> *Note, when attempting to run this command you may be prompted:* Click on `cancel` instead of moving it to trash.
+To allow execution, go to `System Preferences -> Security & Privacy -> General`, and click on `allow`.
+After this, simply repeat the step you prompted for (step 4 or 6). This time, click the `Open` button when prompted.
 
-1. Download the executables for your operating system
-2. Open your favourite terminal, and go to the folder where you download the executables
-3. This will start the node (replace `INSERT_YOUR_ID` with a nickname you choose):
 ```
-subspace-node-x86_64-*-snapshot --chain testnet --wasm-execution compiled --execution wasm --bootnodes "/dns/farm-rpc.subspace.network/tcp/30333/p2p/12D3KooWPjMZuSYj35ehced2MTJFf95upwpHKgKUrFRfHwohzJXr" --rpc-cors all --rpc-methods unsafe --ws-external --validator --telemetry-url "wss://telemetry.polkadot.io/submit/ 1" --name INSERT_YOUR_ID
+./subspace-node-x86_64-*-snapshot \
+  --chain testnet \
+  --wasm-execution compiled \
+  --execution wasm \
+  --bootnodes "/dns/farm-rpc.subspace.network/tcp/30333/p2p/12D3KooWPjMZuSYj35ehced2MTJFf95upwpHKgKUrFRfHwohzJXr" \
+  --rpc-cors all \
+  --rpc-methods unsafe \
+  --ws-external \
+  --validator \
+  --telemetry-url "wss://telemetry.polkadot.io/submit/ 1" \
+  --name INSERT_YOUR_ID
+```
+5. You should see something similar as the output in the terminal.
 ```
-4. After running this command, Windows may ask you for permissions related to firewall, select `allow` in this case.
-5. This will start the farmer (do this in another terminal):
+
 ```
-subspace-farmer-x86_64-*-snapshot farm
+7. We will then open another terminal, change to the downloads directory, then start the farmer node with the following command:
 ```
+./subspace-farmer-x86_64-*-snapshot farm
+```
+7. It may prompt again in here. Refer to the note on step 4.
+</p>
+</details>
 
-### Important Notes!
+## 🤔Notes
 
-***Identity Management***: If you would like to import your polkadot.js wallet so, you can interact with the network, you may do so via
+### Identity Management: 
+If you would like to import your polkadot.js wallet so, you can interact with the network, you may do so via
```

Comment:
This actually needs a warning that running multiple nodes with the same seed WILL cause issues.